scripts/kind-e2e/e2e.sh

```diff
@@ -382,97 +394,48 @@ if [[ $4 = true ]]; then
+    for i in 1 2 3; do
       context=cluster$i
       kubectl config use-context $context
-
-      # Create CRDs required as prerequisite submariner-engine
-      # TODO: Eventually OLM should handle this
-      create_subm_endpoints_crd
-      verify_endpoints_crd
-      create_subm_clusters_crd
-      verify_clusters_crd
-
-      # Add SubM gateway labels
       add_subm_gateway_label
-      # Verify SubM gateway labels
-      verify_subm_gateway_label
-
-      # Deploy SubM Operator
-      deploy_subm_operator
-      # Verify SubM CRD
-      verify_subm_crd
-      # Verify SubM Operator
-      verify_subm_operator
-      # Verify SubM Operator pod
-      verify_subm_op_pod
-      # Verify SubM Operator container
-      verify_subm_operator_container
-
-      # FIXME: Rename all of these submariner-engine or engine, vs submariner
-      # Create SubM CR
-      create_subm_cr
-      # Deploy SubM CR
-      deploy_subm_cr
-      # Verify SubM CR
-      verify_subm_cr
-      # Verify SubM Engine Deployment
-      verify_subm_engine_deployment
-      # Verify SubM Engine Pod
-      verify_subm_engine_pod
-      # Verify SubM Engine container
-      verify_subm_engine_container
-      # Verify Engine secrets
-      verify_subm_engine_secrets
-
-      # Verify SubM Routeagent DaemonSet
-      verify_subm_routeagent_daemonset
-      # Verify SubM Routeagent Pods
-      verify_subm_routeagent_pod
-      # Verify SubM Routeagent container
-      verify_subm_routeagent_container
-      # Verify Routeagent secrets
-      verify_subm_routeagent_secrets
+      subctl join --kubeconfig ${PRJ_ROOT}/output/kind-config/dapper/kind-config-$context \
+                  --clusterid ${context} \
+                  --repository ${subm_engine_image_repo} \
+                  --version ${subm_engine_image_tag} \
+                  --nattport ${ce_ipsec_nattport} \
+                  --ikeport ${ce_ipsec_ikeport} \
+                  --colorcodes ${subm_colorcodes} \
+			            --disable-nat \
+                  broker-info.subm
+
     done
 
     deploy_netshoot_cluster2
     deploy_nginx_cluster3
+
+    # subctl wants a gateway node labeled, or it will ask, but this script is not interactive,
+    # and E2E expects cluster1 to not have the gateway configured at start, so we remove it
+    context=cluster1
+    kubectl config use-context $context
+    del_subm_gateway_label
+
 elif [[ $5 = helm ]]; then
     helm=true
     setup_cluster2_gateway
     setup_cluster3_gateway
-    for i in 2 3; do
-      context=cluster$i
-      kubectl config use-context $context
-
-      # The Helm deploy doesn't respect namespace config, hardcode to what it uses
-      subm_ns=submariner
-
-      verify_subm_engine_deployment
-      verify_subm_engine_pod
-      verify_subm_routeagent_daemonset
-      verify_subm_routeagent_pod
-      verify_subm_engine_container
-      verify_subm_routeagent_container
-      verify_subm_engine_secrets
-      verify_subm_routeagent_secrets
```

Comment:
Why did you remove the verifying functions, shouldn't we check that subctl did everything ?